sos/plugins/openstack_novajoin.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018 Red Hat, Inc., David Vallee Delisle <dvd@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
```

Comment:
You don't seem to be using this anywhere - if it's not needed, best to just drop it.